fastlane/Constants/Constant.swift

```diff
@@ -37,6 +37,17 @@ enum Constant {
     static let projectPath: String = "./\(projectName).xcodeproj"
     static let testOutputDirectoryPath = "./fastlane/test_output"
 
+    // MARK: Platform
+
+    static var platform: PlatformType {
+        if EnvironmentParser.bool(key: "BITRISE_IO") {
+            return .bitriseIO
```

Comment:
I think `bitrise` is enough 🤔

Reply:
Updated in [65d075b](https://github.com/nimblehq/ios-templates/pull/384/commits/65d075ba2b8c6d58128a03c56f81da2c68c3f774)